molecule/eval/converge.yml

```diff
@@ -217,6 +217,50 @@
               "--- {'other': 'cute-table'}\n...\n",
           ]
 
+# EVAL FROM STRING (with retries)
+
+- name: 'Eval with retries'
+  hosts: cluster
+  become: true
+  become_user: root
+  gather_facts: false
+  tasks:
+    - name: Eval returns nil, err
+      import_role:
+        name: ansible-cartridge
+      ignore_errors: true
+      vars:
+        cartridge_scenario:
+          - eval
+        cartridge_eval_body: return nil, "I am horrible error"
+        cartridge_eval_with_retries: true
+        cartridge_eval_retries: 2
+        cartridge_eval_delay: 1
+
+    - name: 'Check eval failed'
```

Comment:
It test is false positive. Let's check count of retries

Reply:
`It test` -> `This test` =)